regress/core/sql-mm-compoundcurve.sql

```diff
@@ -240,3 +240,25 @@ SELECT 'minpoints issues - pass', encode(ST_AsBinary(ST_GeomFromText('COMPOUNDCU
 SELECT 'minpoints issues - fail', ST_GeomFromText('COMPOUNDCURVE(CIRCULARSTRING(0 0,1 1))');
 SELECT 'minpoints issues - fail', ST_GeomFromText('COMPOUNDCURVE(CIRCULARSTRING(0 0))');
 SELECT 'minpoints issues - fail', ST_GeomFromText('COMPOUNDCURVE((0 0),(0 0,1 1))');
+
+-- https://trac.osgeo.org/postgis/ticket/5361
+WITH f(geom) AS (
+SELECT
+  'COMPOUNDCURVE(
+    LINESTRING(2 2, 2.5 2.5),
+    CIRCULARSTRING(2.5 2.5, 4.5 2.5, 3.5 3.5),
+    LINESTRING(3.5 3.5, 2.5 4.5, 3 5, 2 2)
+  )'::geometry AS geom
+)
+SELECT 'curve accessors 01',
+  ST_NumGeometries(geom) as numgeometries,
+  ST_GeometryType(ST_GeometryN(geom,1)) as geometryn_1,
+  ST_GeometryType(ST_CurveN(geom,1)) as curven_1,
+  ST_GeometryN(geom,2) AS geometryn_2,
+  ST_NumCurves(geom) AS numcurves,
+  ST_GeometryType(ST_CurveN(geom,1)) as curven_1,
+  ST_GeometryType(ST_CurveN(geom,2)) as curven_2,
+  ST_GeometryType(ST_CurveN(geom,3)) as curven_3,
+  ST_GeometryType(ST_CurveN(geom,3)) as curven_4,
```

Comment:
this repeats the curven_3 query. Is that indented, or was it rather ST_CurveN(geom,4) ?